api/src/services/users.js

```diff
@@ -551,16 +551,21 @@ const validateUserContact = (data, user, userSettings) => {
   }
 };
 
-const createUserEntities = async (user, appUrl, preservePrimaryContact) => {
+const createUserEntities = async (data, appUrl) => {
```

Comment:
It'd be helpful to add a comment above the function describing what is `data` parameter. Sometimes it's hard to know what to expect without debugging 

```
   * @param {Object} data User information containing: username .........
```

Reply:
It's the same `data` that's documented by the JSDoc for `createUser` and `createUsers` but I'll add it `createUserEntities` to make it clearer